CHANGELOG.md

```diff
@@ -1,6 +1,25 @@
 # Changelog
 
-## [v0.1.0] 
+## 0.2.0 (August 23, 2019)
+
+FEATURES:
+
+* Add support for LBaaS (GH-2)
+* Add support for PaaS (GH-6)
+* Add support for ISO Image Handling (GH-8)
+* Add support for Object Storage (GH-11)
+* Add support for Snapshots (GH-12) and Snapshot Scheduler (GH-13)
+* Add support for Firewall Handling (GH-14)
+
+IMPROVEMENTS:
+
+* Avoid use of fmt.errorf
+* Unit Tests for all functionality
+* Logging support
```

Comment:
Logging was exist before, but we added the capability to switch between Logging level between Debug and Info.

Reply:
yes, that's what I call "logging support" - everything else is just printing stuff.